docs/GettingStarted.md

```diff
@@ -297,3 +297,9 @@ This will result in lines like:
 5984 [003] 03/10/2022-13:56:14.226:{"Message":"Hello, world"}
 ```
 where `5984` is the Process ID in decimal, and `003` is the CPU ID.
+
+To view all trace events from network eBPF extension (`netebpfext.sys`) that would be captured to a file, use the following commands:
```

Comment:
```suggestion
To view all trace events from the network eBPF extension (`netebpfext.sys`) that would be captured to a file, use the following commands:
```

Reply:
Also how would you see all trace events from both ebpfcore.sys and netebpfext.sys?

Reply:
OK updating the readme.